src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyCultureProvider.cs

```diff
@@ -62,7 +63,7 @@ public virtual async ValueTask LoadCurrentCultureResourcesAsync()
             throw new PlatformNotSupportedException("This method is only supported in the browser.");
         }
 
-        var culturesToLoad = GetCultures(CultureInfo.CurrentCulture);
+        var culturesToLoad = GetCultures(CultureInfo.CurrentCulture, CultureInfo.CurrentUICulture);
```

Comment:
In which situation do we need to handle `null` instances. `CultureInfo.CurrentCulture` is never null according to `https://learn.microsoft.com/es-es/dotnet/api/system.globalization.cultureinfo.currentculture?view=net-10.0`. Same for `https://learn.microsoft.com/es-es/dotnet/api/system.globalization.cultureinfo.currentuiculture?view=net-10.0`.

As far as I understand, they are either InvariantCulture or a specific culture like `es-ES` that "walks" up to `InvariantCulture, so the `null` checks should be replaceable against the `InvariantCulture`.